accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/configreader/YamlConfigReader.java

```diff
@@ -293,31 +293,26 @@ protected AuthorizableConfigBean getNewAuthorizableConfigBean() {
         return new AuthorizableConfigBean();
     }
 
-    protected void setupAceBean(final String authorizableId,
-            final Map<String, ?> currentAceDefinition, final AceBean aclBean) {
+    protected void setupAceBean(final String authorizableId, final Map<String, ?> currentAceDefinition, final AceBean aclBean) {
 
         aclBean.setAuthorizableId(authorizableId);
         aclBean.setPrincipalName(authorizableId); // to ensure it is set, later corrected if necessary in
                                                   // AcConfiguration.ensureAceBeansHaveCorrectPrincipalNameSet()
 
-        aclBean.setJcrPath(getMapValueAsString(currentAceDefinition,
-                ACE_CONFIG_PROPERTY_PATH));
+        String jcrPath = getMapValueAsString(currentAceDefinition, ACE_CONFIG_PROPERTY_PATH);
+        String cleanJcrPath =  StringUtils.removeEnd(jcrPath, "/");
+        LOG.debug("cleaned jcr path: {} to {}", jcrPath, cleanJcrPath);
```

Comment:
@luckyluke80 this log is misleading (it will also show if nothing was removed) - I think it's best to just remove the log